PaddlePaddle/PARL
# add api set_params/get_params in Model

- add api set_params/get_params in Network; 
- add unittest of api set_params/get_params;
- add Interface of Network and LayerFunc to solve circular imports; 
- refine parameter_names api in Network, which will ensure order consistent between different instance of same Network;
- run teamcity unittest in both GPU and CPU;
